entrypoint.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+ruby ./helpers/wait_for_documentserver_start.rb
+rspec
```

Comment:
Do you removed ability to run custom `rspec` at all?

Do you plan to make it in seprarte PR (I think it's a great idea)